apps/constellation/constellation.cpp

```diff
@@ -121,6 +121,9 @@ uint16_t LookupLocalPort(Manifest const &manifest, ServiceIdentifier::Type servi
 std::shared_ptr<ledger::DAGInterface> GenerateDAG(std::string const &db_name, bool load_on_start,
                                                   Constellation::CertificatePtr certificate)
 {
+  FETCH_UNUSED(db_name);
```

Comment:
Why is this necessary?

Reply:
It's not, I think I deleted it at the same time you commented ahah